src/components/Button.js

```diff
@@ -1,12 +1,24 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
+let defaultBgColor = {
+  backgroundColor: 'orange'
+};
+
+
 class Button extends Component {
   render() {
+    const wideStyle = (this.props.wide) ? 'wideStyle' : '';
+    let bgColor;
+    if (!this.props.color) {
+      bgColor = defaultBgColor;
+    } else {
+      bgColor = { backgroundColor: this.props.color };
+    }
     return (
-      <div>
-        <button>{this.props.name}</button>
-      </div>
+      <button className={`${wideStyle}`} id='button-component' style={bgColor}>
```

Comment:
The template literal is unnecessary here. Could be just `className={wideStyle}`.